src/author.rs

```diff
@@ -2,6 +2,14 @@
 //!
 //! This module defines the `Author` data structure together with its utility
 //! enum `AuthorParseError`.
+//!
```

Comment:
The commit message should say, which documentation was adjusted.

Reply:
The comma in you comment is grammatically not required.